frontend-employees/src/app/components/vacation-planning/vacation-planning.component.scss

```
@@ -0,0 +1,69 @@
+.planner-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-column-gap: 30px;
+  grid-row-gap: 20px;
+  align-items: center;
+
+  .vacation-days {
+    align-self: center;
+    color: limegreen;
+  }
+
+  .mat-table {
+    grid-column: 1/5;
+  }
+
+  .mat-calendar {
+    min-height: 260px;
+  }
+
+  h3 {
+    grid-column: 1/5;
+  }
+
+  .indicator-container {
+    display: flex;
+
+    .approved-indicator,
+    .pending-indicator {
+      &:before {
+        content: '';
+        display: inline-block;
+        height: 12px;
+        width: 12px;
+        background: limegreen;
+      }
+    }
+
+    .approved-indicator:before {
+      background: limegreen;
+    }
+
+    .pending-indicator {
+      margin-left: 20px;
+
+      &:before {
+        background: dodgerblue;
+      }
+    }
+  }
+
+  ::ng-deep {

```

Mit `::ng-deep` wird das CSS global angewendet. Da dies aber nur für diese Komponente notwendig ist den CSS Scope mit einer Klasse einschränken.
```suggestion
  .vacation-calendar ::ng-deep {
```